8/README.MD

```diff
@@ -558,34 +545,4 @@ They interact as follows:
 
 An **user** sends a request to the **controller**. the **controller** then queries the **model** for data. The **model** then transmits the data to the **view**, which then gets displayed to the **user**.
 
-This paradigm is good for security, as it keeps the user from directly accessing the model. With MVC, whenever a user wants to get info from the model (database) or put info into the model, then it has to go through a controller. As such, when done correctly, all of the interactions with the model are predefined. If we gave the user direct access to the model, then the user could for example delete all data in the model or steal all your users' data.
```

Comment:
MVC is just a design pattern, not a security model. DB is just a vulnerable if your controller has read/write access to it.